### DISCUSSION TITLE
Is there a planned or community dark mode admin interface?

### DISCUSSION THREAD

**Discussion by paulpopus** [strapi/strapi#7201] [Legacy Topics]
I looked around and couldn't find anything on the matter so I'm wondering if this has been discussed or is being discussed somewhere.

**Comment by lauriejim** [comment]
Hello! We will work on a redesign of the admin and introduce a theme system (but not in a short term, it's just in our mind)
So the answer is yes, but not now ^^
You will be able to customize the 5 main colors, and all the admin will be based on these colors.
Can't wait to see that :)
